.github/workflows/tests.yml

```diff
@@ -133,6 +133,17 @@ jobs:
           cat bottles/steps_output.txt
           rm bottles/steps_output.txt
 
+      - name: Upload logs
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: logs (${{ matrix.version }})
+          path: bottles/logs
+
+      - name: Run brew test-bot --only-cleanup-after
```

Comment:
Why the move?

Reply:
We want to remove the logs directory before bottle upload. brew test-bot does this already but I fucked up as it needs `cd bottles` first.

Alternatively, we can handle it in the workflow and make this `rm -rf bottles/log bottles/home`.

Reply:
I'm fine either way!

Reply:
Agh, I forgot this would create a `steps_output.txt` which I will now need to delete somewhere. Maybe `rm` is going to be easier.